### DISCUSSION TITLE
Consistency in naming sqlite DBs

### DISCUSSION THREAD

**Discussion by diwakergupta** [stacks-network/stacks-core#2410] [Ideas]
Admittedly a minor thing: right now we have several different patterns in how we name SQlite files. Would be nice to have some consistency, to make it easier for folks to navigate and understand the persistent state.

```
peer_db.sqlite
mempool.db
chainstate/chain-01000000-mainnet/vm/index
burnchain/db/bitcoin/mainnet/sortition.db/marf
```

**Comment by jcnelson** [comment]
"Sure, the nuclear reactor may have some critical design flaws that put it at risk of a meltdown, but you know what's worth bringing to the developers' attention?  The color of the bike shed."

**Reply by diwakergupta** [reply]
😓 I'm just trying out the Discussions feature as an experiment, don't mind me! One nice thing about discussions (vs. issues) is we can capture, debate (and dismiss) ideas without cluttering the issues.

**Reply by psq** [reply]
to @diwakergupta 's point, not having a proper extension on some files prevents autocompletion on `marf` and `index` so that would be a plus (using `bash_completion`)

**Comment by diwakergupta** [comment]
This was addressed in #2421 , thanks @lgalabru !
